bokeh/models/widgets/icons.py

```diff
@@ -20,6 +20,7 @@ class Icon(AbstractIcon):
 
     """
 
+    # TODO this conflicts with PlotObject.name and should be renamed
```

Comment:
`name` is definitely used and I think we are stuck with it (looking back, maybe `alias` or `tag` would have been better). Regarding this, 'icon_name` or `type` or `kind` seem OK. Or even `icon`